tests/test_intl.py

```diff
@@ -281,7 +299,9 @@ def test_text_glossary_term_inconsistencies(app, warning):
     warnings = getwarning(warning)
     expected_warning_expr = (
         u'.*/glossary_terms_inconsistency.txt:\\d+: '
-        u'WARNING: inconsistent term references in translated message\n')
+        u'WARNING: inconsistent term references in translated message.'
+        u" original: \[':term:`Some term`', ':term:`Some other term`'\],"
+        u" translated: \[':term:`SOME NEW TERM`'\]\n")
```

Comment:
You need to add `u` prefix to expected regexp like `u?':term:...'` for py27. Please check error logs on travis:
https://travis-ci.org/sphinx-doc/sphinx/jobs/284722614#L2006